leptonai/photon/photon.py

```diff
@@ -174,7 +176,10 @@ class Photon(BasePhoton):
     image: str = BASE_IMAGE
 
     # The args for the base image.
-    args: list = BASE_IMAGE_ARGS
+    args: List[str] = BASE_IMAGE_ARGS
+    cmd: Optional[List[str]] = BASE_IMAGE_CMD
```

Comment:
Question for @xiang90 @ccding - is it more common to call it "cmd" or "entry_point" in the cloud native context? Python doesn't really have a strong convention so I am kinda curious what name is the best.